lhotse/cut.py

```diff
@@ -42,56 +42,68 @@ class Cut:
     """
     id: str
 
-    # Begin and duration are needed to specify which chunk of features to load.
+    # Begin and duration are needed to specify which chunk of features/recording to load.
     start: Seconds
     duration: Seconds
 
+    # Supervisions that will be used as targets for model training later on. They don't have to cover the whole
+    # cut duration. They also might overlap.
+    supervisions: List[SupervisionSegment]
+
     # The features can span longer than the actual cut - the Features object "knows" its start and end time
     # within the underlying recording. We can expect the interval [begin, begin + duration] to be a subset of the
     # interval represented in features.
-    features: Features
+    features: Features = None
 
-    # Supervisions that will be used as targets for model training later on. They don't have to cover the whole
-    # cut duration. They also might overlap.
-    supervisions: List[SupervisionSegment]
+    # For the cases that the model was trained by raw audio instead of features
+    recording: Recording = None
 
     @property
     def channel(self) -> int:
-        return self.features.channel_id
+        # Return the first channel's id
+        return self.features.channel_id if self.features is not None else self.recording.channel_ids[0]
 
     @property
     def recording_id(self) -> str:
-        return self.features.recording_id
+        return self.features.recording_id if self.features is not None else self.recording.id
 
     @property
     def end(self) -> Seconds:
         return self.start + self.duration
 
     @property
     def frame_shift(self) -> Seconds:
-        return self.features.frame_shift
+        return self.features.frame_shift if self.features is not None else 0
```

Comment:
when there's no features I'd rather return `None` and change the type to `Optional[Seconds]`; returning 0 might lead to silent bugs in some cases